maps/tether/tether-05-station1.dmm

```diff
@@ -24233,7 +24233,7 @@
 /area/gateway/prep_room)
 "kHf" = (
 /obj/machinery/conveyor_switch/oneway{
-	convdir = -1;
+	//convdir = -1;		//Incompatable with new behavior -Reo
```

Comment:
I feel this change should really, really be tested by hosting the tether map and going to this conveyor.

Reply:
I have no idea where this belt is-

Reply:
I found it, and it uses belts which could easily be reversed. it's also linked to literally two belts, in an area that i've rarely seen used
![image](https://user-images.githubusercontent.com/84661000/233167613-f71707a7-2272-4e40-bb75-e2d352df31b3.png)

Reply:
the belt setup also seems to be a bit busted, probably because one belt goes in the wrong direction in reverse while the other doesnt. are conveyor corners only available in one direction?

Reply:
REGARDLESS- the issue it causes is nonexistent, they can still just switch the switch twice if the singular belt being used is important to be used

Reply:
That alleviates concerns on this part!

Reply:
Ah yes, the surface cargo shop. Replaced the old surface mining area when that hardware was all moved up to station 1 with cargo.

I'd recommend just deleting the offending line from the .dmm though, those are map files that are basically never read by anything except a map editing program and the commented out line (+comment) will either go completely unseen, or worse, the editor might try to strip the comments out later and accidentally break stuff.